compiler/rustc_codegen_cranelift/src/base.rs

```diff
@@ -515,7 +515,7 @@ fn codegen_fn_body(fx: &mut FunctionCx<'_, '_, '_>, start_block: Block) {
             }
             TerminatorKind::Drop { place, target, unwind: _, replace: _ } => {
                 let drop_place = codegen_place(fx, *place);
-                crate::abi::codegen_drop(fx, source_info, drop_place);
+                crate::abi::codegen_drop(fx, source_info, drop_place, *target);
 
                 let target_block = fx.get_block(*target);
                 fx.bcx.ins().jump(target_block, &[]);
```

Comment:
Maybe move this jump into codegen_drop?

Reply:
Do you think that's an improvement? It seems to me that it's fairly natural for the codegen to "fall through" into this jump -- even if sometimes it early jumps out.

(Happy to do it either way but it feels like it makes things worse to me.)

Reply:
For future unwinding support this will be necessary anyway as in that case the call to the drop glue will be a terminator too.